### PR TITLE
Correct return type of Node querystring.parse

### PR DESCRIPTION
**Pull request by OliverJAsh**
https://nodejs.org/api/querystring.html#querystring_querystring_parse_str_sep_eq_options

```
> require('querystring').parse('foo')
{ foo: '' }
```

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/node/index.d.ts [hunk 1]
```diff
@@ -797,7 +797,7 @@ declare module "querystring" {
     }
 
     export function stringify<T>(obj: T, sep?: string, eq?: string, options?: StringifyOptions): string;
-    export function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): any;
+    export function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): { [key: string]: string };
```
Comment (Flarna) [inline types/node/index.d.ts]: I think the return type should be `{ [key: string]: string | string[] }` as `'foo=bar&abc=xyz&abc=123'` is parsed into
```
{
  foo: 'bar',
  abc: ['xyz', '123']
}
```
according to https://nodejs.org/api/querystring.html#querystring_querystring_parse_str_sep_eq_options

**Review by Flarna — CHANGES_REQUESTED** [review]

File: types/node/index.d.ts (hunk 1 above)
Comment (OliverJAsh) [reply types/node/index.d.ts]: Thanks, pushed a fix!

**Comment by OliverJAsh** [comment]
PR build timed out. Who can restart? I can't seem to…

**Review by Flarna — APPROVED** [review]

**Comment by OliverJAsh** [comment]
Do I have to manually ping a reviewer for express? @Flarna

**Comment by andy-ms** [comment]
No, maintainers should look at all PRs without needing to be pinged.

**Pull request merged**